src/shipit_code_coverage/shipit_code_coverage/uploader.py

```diff
@@ -69,8 +69,17 @@ def get_latest_codecov():
 
 
 def codecov_wait(commit):
```

Comment:
This looks good? It ensures `retry` doesn't stop looping right away when `check_codecov_job` returns False (preserving `wait_until`'s behavior).

Reply:
Yes, this is fine!